pandas/core/generic.py

```diff
@@ -8830,6 +8830,12 @@ def mask(self, cond, other=np.nan, inplace=False, axis=None, level=None,
             extend the index when shifting and preserve the original data.
         axis : {0 or 'index', 1 or 'columns', None}, default None
             Shift direction.
+            
+        .. versionchanged:: 0.24.0
```

Comment:
Formatting is a bit off. The version added needs to go at the end. Something like

    fill_value : object, optional
        the scalar value to use for newly introduced missing values.
        the default depends on the dtype of `self`. for numeric data,
        ``np.nan`` is used. for datelike, ``pandas.nat`` is used.
    
        .. versionadded:: 0.24.0

Reply:
done